src/platforms/rcore_desktop_glfw.c

```diff
@@ -1,6 +1,6 @@
 /**********************************************************************************************
 *
-*   rcore_desktop - Functions to manage window, graphics device and inputs
```

Comment:
This is due to this source file being previously named ``rcore_desktop.c`` (without ``_glfw``).